containers/ecr-viewer/guides/technical-acceptance-testing.md

```diff
@@ -0,0 +1,257 @@
+
+## Prerequisites
+
+- You’ve installed DIBBs services using [DIBBs AWS Terraform,](https://github.com/CDCgov/dibbs-aws) [DIBBs Azure Terraform](https://github.com/CDCgov/dibbs-azure), [DIBBs VM](https://github.com/CDCgov/dibbs-vm), or a custom installation.
+- You’ve set the appropriate environment variables for your configuration based on the [setup guide](https://cdcgov.github.io/dibbs-ecr-viewer/documents/Setup_Guide.html#environment-variable-setup).
+- You have access to sample eCR/RR files.
+
+## Acceptance Checklist
+
+The following instructions will guide you through this list. Mark the checklist as you complete each step.
+
+- Verified that the eCR Viewer health check is available
+- Verified that the metadata database is up to date (if using one)
+- Verified that the process-zip endpoint run was successful
+- Verified that the eCR FHIR bundle is available in blob storage
+- Verified that the processed eCR is visible in the viewer
+
+## Installation Testing
+
+To test the installation, we’ll run a series of API requests to the DIBBs services.
+
+`{{dibbs-url}}` will be used throughout the document to indicate the URL your services are available on — use your own URL instead.
+
+Instructions have been provided for different operating systems, so follow the instructions applicable to your machine.
+
+### 1. eCR Viewer Health Check
+
+Here, we’ll verify that the eCR Viewer service is available to receive requests by running the eCR Viewer Health Check request.
+
+<details>
+<summary>Powershell (compatible with versions 5 and 7)</summary>
+
+```powershell
+Invoke-WebRequest -Uri "https://{{dibbs-url}}/ecr-viewer/api/health-check" -UseBasicParsing
+```
+
+</details>
+
+<details>
+<summary>Unix (Mac, Linux) or Windows Command Prompt</summary>
+
+```bash
+curl https://{{dibbs-url}}/ecr-viewer/api/health-check
+```
+
+</details>
+
+You should receive the following response:
+
+```json
+{ "status": "OK", "version": {{app version}}, "dependencies": { {{dependency name}}: {{status ("UP" or "DOWN")}} } }
+```
+
+### 2. Database Migrations
+
+If you are running the NBS-integrated version of the Viewer, you can skip this step.
+
+If you are running the viewer with a metadata database (standalone or dual boot), you will need to run database migrations upon the first install of the application or anytime there is an update to the database schema. If migrations need to be applied, you will see an error when trying to use the viewer.
+
+![Screenshot of the DIBBs eCR Viewer with an error message "eCR Viewer setup is incomplete"](./assets/setup-incomplete.png)
+
+In order to apply migrations, you will need the migration secret. This can be set to a static value by setting the `METADATA_DATABASE_MIGRATION_SECRET`, otherwise it will be generated randomly each time the viewer is started. Once generated, it will be logged to the eCR Viewer server logs.
```

Comment:
```suggestion
In order to apply migrations, you will need the migration secret. This can be set to a static value by setting the `METADATA_DATABASE_MIGRATION_SECRET`, otherwise it will be generated randomly each time the viewer is started. Once generated, it will be logged to the eCR Viewer server logs. Check your container's logs to find this secret - it will be logged as `<$whatever the actual log string is>`.
```